client/src/components/Maps/Directions.jsx

```diff
@@ -1,185 +1,119 @@
 import React, { useState } from "react";
-import {
-  Alert,
-  Button,
-  Col,
-  Divider,
-  Form,
-  Input,
-  Layout,
-  Menu,
-  Row,
-  Space,
-  Spin,
-  Steps,
-} from "antd";
+import { Col, Divider, Layout, Radio, Row, Steps } from "antd";
 import MapView from "./MapView";
-import nominatimService from "../../services/nominatim/nominatim-service";
 import { Content } from "antd/es/layout/layout";
-import Sider from "antd/es/layout/Sider";
+import AddressSetter from "./AddressSetter";
+
 const Directions = () => {
-  const { searchNominatim } = nominatimService();
   const [mapParams, setMapParams] = useState({});
   const [mapMode, setMapMode] = useState("userLoc");
-
-  // Spinners
-  const [loadingSpinner, setLoadingSpinner] = useState(false);
-
   const [currentStep, setCurrentStep] = useState(0);
 
-  // Alerts
-  const [showAlert, setShowAlert] = useState(false);
+  const [source, setSource] = useState();
+  const [destination, setDestination] = useState();
 
-  const [menuResults, setMenuResults] = useState([]);
-  const [selectedMenuId, setSelectedMenuId] = useState("");
-
-  const handleMenuClick = (e) => {
-    console.log("onClick");
```

Comment:
I guess you will remove console logs in future or can comment for now and use in dev before merging?

Reply:
Done